trainee_template/synthesis.amlg

```diff
@@ -84,8 +84,12 @@
 			constrained_features (null)
 		))
 
-		;if global residuals have have not been calculated yet, recalculate them. they are needed regardless of which residual mode is being used
-		(if (not (get hyperparam_map "allFeatureResidualsCached"))
+		;if global residuals have have not been calculated yet, recalculate them only if there are enough training cases
+		;they are needed regardless of which residual mode is being used
+		(if (and
+				(not (get hyperparam_map "allFeatureResidualsCached"))
+				(>= (call GetNumTrainingCases) 2)
```

Comment:
Why 2?

Reply:
Need two to make a prediction.

Reply:
2 is minimum to predict and compute residuals. With that the PR title change, this makes more sense.